nmxact/xact/image.go

```diff
@@ -68,7 +69,7 @@ func (r *ImageUploadResult) Status() int {
 }
 
 func buildImageUploadReq(imageSz int, hash []byte, upgrade bool, chunk []byte,
-	off int) *nmp.ImageUploadReq {
+	off int, image_num int) *nmp.ImageUploadReq {
```

Comment:
Nitpick: Should this use camel case instead?

Reply:
Oops, all occurrences of snake case where fixed!

Reply:
Fixed.